### ISSUE TITLE
[BUG Report]: Repeated progress bar in Model.fit()

### ISSUE THREAD

**Issue by Tvvgge** [SciSharp/TensorFlow.NET#1121]
### Description

In Tensorflow.Keras 0.10.5 the progress bar look like this:
Epoch: 001/100
0009/0009 [=========================>...] - 2ms/step - loss: 20.894537 - val_loss: 31.065659
Epoch: 002/100
0009/0009 [=========================>...] - 4ms/step - loss: 19.879198 - val_loss: 30.460979
Epoch: 003/100
0009/0009 [=========================>...] - 2ms/step - loss: 18.928097 - val_loss: 29.738377

When upgraded to 0.11.0 the progress bar becomes:
Epoch: 001/100
Epoch: 001/000======================>...] - 6ms/step - loss: 20.9055106
0009/0009 [=========================>...] - 6ms/step - loss: 20.905510 - val_loss: 30.871361
Epoch: 002/100
Epoch: 001/000======================>...] - 2ms/step - loss: 19.939381
0009/0009 [=========================>...] - 2ms/step - loss: 19.939381 - val_loss: 30.301033
Epoch: 003/100
Epoch: 001/000======================>...] - 2ms/step - loss: 18.945526
0009/0009 [=========================>...] - 2ms/step - loss: 18.945526 - val_loss: 29.720556

### Reproduction Steps

Upgrade the Tensorflow.Keras to version 0.11.0

**Comment by Wanglongzhi2001** [comment]
Thanks, I will fix it later.

**Comment by Wanglongzhi2001** [comment]
Hello, I have fixed this bug. ^_^
